esp32_s3_lcd_ev_board/sub_board/sub_board_800x480.c

```diff
@@ -0,0 +1,172 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: CC0-1.0
+ */
+
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "freertos/semphr.h"
+#include "esp_lcd_panel_io.h"
+#include "esp_lcd_panel_ops.h"
+#include "esp_lcd_panel_rgb.h"
+#include "esp_lcd_touch_gt1151.h"
+#include "esp_log.h"
+
+#include "sdkconfig.h"
+#include "../priv_include/bsp_err_check.h"
+#include "../priv_include/bsp_sub_board.h"
```

Comment:
Please define the `priv_include` directory as private include in CMakeLists.txt so we can get rid of the relative `..` in path

Reply:
Fixed, PTAL [1](https://github.com/espressif/esp-bsp/pull/89/commits/8723853de84500501a915f9a2c568d9e528a8b67#diff-43098840b47cfd6e4dfb4db2d0ff0af2f2ba8569a38a096ebc253347cc4e328eL17-R18), [2](https://github.com/espressif/esp-bsp/pull/89/commits/abc7099df435b938ce2cd02f1ecafc7ca546c79c#diff-d6e640338b4dc95a1d9e90b81b17d1ac1a0d001f50977d8a52be5e2895838044R19-R20) and [3](https://github.com/espressif/esp-bsp/pull/89/commits/abc7099df435b938ce2cd02f1ecafc7ca546c79c#diff-521a0f3ed3e2e3565f9da746eb3de4eed006957460f7cd6077b1491d48d90f60R23-R24).